pages/events/[id].js

```diff
@@ -12,20 +16,88 @@ function EventDetail() {
     return <h1>hmm... no event yet 🤔?</h1>;
   }
 
-  const { title, startTime, endTime, location, creator, isParticipant } =
-    currentEvent;
+  const { title, startTime, endTime, location, creator } = currentEvent;
+
+  function handleVoteEmoji(voteEmoji) {
+    if (emojiCheckmark === "") {
+      setEmojiCheckmark(voteEmoji);
+    } else {
+      setEmojiCheckmark("");
+    }
+  }
 
   return (
-    <>
+    <StyledWrapper>
       <h2>{title}</h2>
-      <span>{startTime}</span>
-      <span>{endTime}</span>
-      <p>Location: {location}</p>
-      <p>Creator: {creator}</p>
-      <p>Are you there? {isParticipant ? "Yes" : "No"}</p>
-      <Link href="/events">Go Back</Link>
-    </>
+      <StyledEmojiCheckmark>{emojiCheckmark}</StyledEmojiCheckmark>
+      <StyledCardContainer>
+        <span>{startTime}</span>
+        <span>{endTime}</span>
+        <p>Location: {location}</p>
+        <p>Creator: {creator}</p>
+      </StyledCardContainer>
+      <StyledVoteButtonContainer>
+        {["👍", "❓", "👎"].map((voteEmoji) => {
+          return (
+            <StyledVoteButtons
+              onClick={() => handleVoteEmoji(voteEmoji)}
+              key={voteEmoji}
+            >
+              <span aria-label="emoji">{voteEmoji}</span>
+            </StyledVoteButtons>
+          );
+        })}
+      </StyledVoteButtonContainer>
+      <Link href="/events">
+        <StyledLink>↩︎</StyledLink>
+      </Link>
```

Comment:
you are nesting an `a`-tag within another `a`-tag. If you want to style a `Link` component, you can do so by extending that component. See [this as reference](https://github.com/neuefische/web-hh-22-7/blob/main/sessions/react-styled-components/react-styled-components.md#styling-a-custom-component).

This part of your code should look like this:
```suggestion
      <StyledLink href="/events">↩︎</StyledLink>
```